exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/metrics_exporter/__init__.py

```diff
@@ -51,58 +51,70 @@
 )
 from opentelemetry.sdk.metrics.export import (
     MetricRecord,
-    MetricsExporter,
+    PassthroughMetricsExporter,
     MetricsExportResult,
 )
+from opentelemetry.sdk.metrics.export.aggregate import (
+    HistogramAggregator,
+    LastValueAggregator,
+    MinMaxSumCountAggregator,
+    SumAggregator,
+    ValueObserverAggregator,
+)
 
 logger = logging.getLogger(__name__)
 DataPointT = TypeVar("DataPointT", IntDataPoint, DoubleDataPoint)
 
 
 def _get_data_points(
-    sdk_metric: MetricRecord, data_point_class: Type[DataPointT]
+    sdk_metric_record: MetricRecord, data_point_class: Type[DataPointT]
 ) -> List[DataPointT]:
 
-    data_points = []
-
-    for (
-        label,
-        bound_counter,
-    ) in sdk_metric.instrument.bound_instruments.items():
-
-        string_key_values = []
-
-        for label_key, label_value in label:
-            string_key_values.append(
-                StringKeyValue(key=label_key, value=label_value)
-            )
-
-        for view_data in bound_counter.view_datas:
-
-            if view_data.labels == label:
-
-                data_points.append(
-                    data_point_class(
-                        labels=string_key_values,
-                        value=view_data.aggregator.current,
-                        start_time_unix_nano=(
-                            view_data.aggregator.last_checkpoint_timestamp
-                        ),
-                        time_unix_nano=(
-                            view_data.aggregator.last_update_timestamp
-                        ),
-                    )
-                )
-                break
-
-    return data_points
+    from ipdb import set_trace
+    set_trace()
+
+    # FIXME: use the exporter kind property to set the start_time_unix_nano
+    # and time_unix_nano accordingly
+
+    if isinstance(sdk_metric_record.aggregator, SumAggregator):
+        value = sdk_metric_record.aggregator.checkpoint
+        start_time_unix_nano = sdk_metric_record.start
```

Comment:
Are `start `and `end` fields the right values? `process_start` and `process_end`?